### PR TITLE
fix return type in odsp doclib utils

### PR DESCRIPTION
**Pull request by jatgarg**

### CODE REVIEW

File: packages/utils/odsp-doclib-utils/src/odspRequest.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export async function getAsync(
     url: string,
     authRequestInfo: IOdspAuthRequestInfo,
 ): Promise<Response> {
-    return authRequest(authRequestInfo, async (config: RequestInit) => fetch(url, config) as Response);
+    return authRequest(authRequestInfo, async (config: RequestInit) => fetch(url, config) as Promise<Response>);
```
Comment (vladsud) [inline packages/utils/odsp-doclib-utils/src/odspRequest.ts:16]: can we just drop any casting here and let compiler figure out the type?
If not, does it point to a need to fix it in some other place?

**Review by vladsud — APPROVED** [review]

File: packages/utils/odsp-doclib-utils/src/odspRequest.ts (hunk 1 above)
Comment (jatgarg) [reply packages/utils/odsp-doclib-utils/src/odspRequest.ts:16]: Yes we can drop it. Let me add lint disbale rule then.

File: packages/utils/odsp-doclib-utils/src/odspRequest.ts [hunk 2]
```diff
@@ -13,7 +13,8 @@ export async function getAsync(
     url: string,
     authRequestInfo: IOdspAuthRequestInfo,
 ): Promise<Response> {
```
Comment (markfields) [inline packages/utils/odsp-doclib-utils/src/odspRequest.ts:15]: Shouldn't this update to `Promise<any>`?

Also, next time please provide an initial comment/description on the PR and on the commit for context for others who look at this PR or commit in the future.

Comment (markfields) [reply packages/utils/odsp-doclib-utils/src/odspRequest.ts:15]: @jatgarg

Comment (jatgarg) [reply packages/utils/odsp-doclib-utils/src/odspRequest.ts:15]: In driver too, we directly cast this to Response as we expect it to be a Response object.
Yes sure for the description thing. It was small so I thought the heading is sufficient.

**Pull request merged**